tests/sentry/api/test_id_or_slug_path_parameters.py

```diff
@@ -278,6 +322,61 @@ def organization_test(self, endpoint_class, slug_params, *args):
 
         self.assert_conversion(endpoint_class, converted_slugs, converted_ids)
 
+    @patch("sentry.api.bases.organization.OrganizationEndpoint.check_object_permissions")
```

Comment:
Is it also possible to override the base [`check_object_permissions`](https://github.com/encode/django-rest-framework/blob/master/rest_framework/views.py#L339) and use the fixture to avoid having to this endpoint by endpoint?

Reply:
You can use the same fixture to do this and include two things in the `with`